pkg/systemd/graphs.scss

```diff
@@ -4,24 +4,13 @@
 @import "../../node_modules/@patternfly/patternfly/components/Page/page.css";
```

Comment:
As I said above, I think these should move to their .scss counterparts, so that variables actually work.